pyproject.toml

```diff
@@ -57,6 +57,7 @@ dev-dependencies = [
 	"sphinx-autoapi>=3.0.0; python_version>='3.9'",
 	"sphinx-rtd-theme>=2.0.0",
 	"types-pillow>=10.2.0.20240822",
+	"pymongo>=4.10.1"
```

Comment:
This isn't a `dev-dependency`, but rather an ACTUAL `dependency` and should be moved up into that section